components/director/internal/repo/create.go

```diff
@@ -46,7 +48,7 @@ func (c *universalCreator) Create(ctx context.Context, dbEntity interface{}) err
 	_, err = persist.NamedExec(stmt, dbEntity)
 	if pqerr, ok := err.(*pq.Error); ok {
 		if pqerr.Code == persistence.UniqueViolation {
-			return &notUniqueError{}
+			return apperrors.NewNotUniqueError("")
```

Comment:
Maybe we could pass `pqerr.Column` to this constructor to get more detailed errors?